src/Hedgehog/Extras/Test/Process.hs

```diff
@@ -32,7 +32,7 @@ import           Data.Eq
 import           Data.Function
 import           Data.Int
 import           Data.Maybe (Maybe (..))
-import           Data.Semigroup ((<>))
+import           Data.Monoid
```

Comment:
Probably better if that had an explicit import list. Especially if its only importing `<>` .

Reply:
Added explicit import list.